tasks/php.js

```diff
@@ -19,30 +20,41 @@ module.exports = function (grunt) {
 			}, function (res) {
 				var statusCodeType = Number(res.statusCode.toString()[0]);
 				if ([2, 3, 4].indexOf(statusCodeType) !== -1) {
-					return cb();
+					if (!serverStarted) {
+						serverStarted = true;
+						done(true);
+					}
+					return;
 				} else if (statusCodeType === 5) {
 					grunt.fail.warn(
 						'Server docroot returned 500-level response. Please check ' +
 						'your configuration for possible errors.'
 					);
-					return cb();
+					if (!serverStarted) {
+						serverStarted = true;
+						done(true);
+					}
```

Comment:
Guys, I get it.
But I just fixed a bug, then you and I wasted some time on formatting back-and-forth, so I automated it (one line in .jscsrc). That way next time, everyone saves time.
Same spirit as submitting my PR: win for everyone, bonus continual improvement.
Now if you don't want it, don't take it.